query! macro produces invalid code for TIME[] arrays

The following code statement produces a compile error, where `times` is a `Vec<chrono::NaiveTime>`:

```rust
query!("SELECT UNNEST($1::TIME[])", &times);
```

The compile error is

```
error[E0433]: failed to resolve: could not find `sqlx` in `types`
  --> src/routing/submission.rs:87:2
   |
87 |     query!("SELECT UNNEST($1::TIME[])", &times);
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ could not find `sqlx` in `types`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

warning: unused import: `&`
  --> src/routing/submission.rs:87:38
   |
87 |     query!("SELECT UNNEST($1::TIME[])", &times);
   |                                         ^
   |
   = note: `#[warn(unused_imports)]` on by default

error: aborting due to previous error; 1 warning emitted

For more information about this error, try `rustc --explain E0433`.
```

Running `cargo expand`, this seems to be the problematic part:

```rust
                        let ty_check = sqlx::ty_match::WrapSame::<
                            &[sqlx::types::sqlx::types::chrono::NaiveTime],
                            _,
                        >::new(&_expr)
                        .wrap_same();
```

From my `Cargo.toml` file:

```toml
sqlx = { version = "0.4.1", features = ["runtime-tokio-rustls", "postgres", "macros", "chrono"], default-features = false }
```

--- a/sqlx-macros/src/database/postgres.rs
+++ b/sqlx-macros/src/database/postgres.rs
@@ -80,7 +80,7 @@ impl_database_ext! {
         Vec<sqlx::types::Uuid> | &[sqlx::types::Uuid],
 
         #[cfg(feature = "chrono")]
-        Vec<sqlx::types::chrono::NaiveTime> | &[sqlx::types::sqlx::types::chrono::NaiveTime],
+        Vec<sqlx::types::chrono::NaiveTime> | &[sqlx::types::chrono::NaiveTime],
 
         #[cfg(feature = "chrono")]
         Vec<sqlx::types::chrono::NaiveDate> | &[sqlx::types::chrono::NaiveDate],

